plugin/build.gradle.kts

```diff
@@ -36,19 +36,55 @@ tasks.withType<PublishTask> {
     channels(channel)
 }
 
-val langs = setOf("en", "ru", "fr", "de", "pl", "it", "zh", "ja", "uk", "el", "ro", "es", "pt", "sk", "fa", "nl")
+fun ExternalModuleDependency.excludesForLT() {
+    exclude("org.slf4j", "slf4j-api")
+    //exclude("ml.dmlc", "xgboost4j")
+    exclude("com.typesafe.akka")
+    exclude("org.scala-lang")
+    exclude("biz.k11i", "xgboost-predictor")
+
+    exclude("com.google.guava", "guava")
+    exclude("com.intellij", "annotations")
+    exclude("net.java.dev.jna", "jna")
+
+    exclude("javax.xml.bind", "jaxb-api")
+    exclude("org.glassfish.jaxb", "jaxb-runtime")
+
+    exclude("com.fasterxml.jackson.core", "jackson-databind")
+
+    exclude("org.apache.lucene", "lucene-core")
+    exclude("org.apache.lucene", "lucene-backward-codecs")
+}
 
 dependencies {
     compileOnly(kotlin("stdlib"))
 
     compile("org.languagetool", "languagetool-core", Versions.languageTool) {
+        excludesForLT()
+    }
+
+    compile("org.languagetool", "language-en", Versions.languageTool) {
+        excludesForLT()
+    }
+
+    testRuntime("org.languagetool", "language-ru", Versions.languageTool) {
+        excludesForLT()
+    }
+
+    // region aether-for-non-idea-ide
```

Comment:
I would even abstract it into `autherDependencies` function

Reply:
Done